docs/sources/panels/_index.md

```diff
@@ -4,4 +4,25 @@ type = "docs"
 [menu.docs]
 identifier = "panels"
 weight = 4
-+++
\ No newline at end of file
++++
+
+# Panel overview
+
+The *panel* is the basic visualization building block in Grafana. Each panel has a query editor specific to the data source selected in the panel. The query editor allows you to extract the perfect visualization to display on the panel.
+
+There are a wide variety of styling and formatting options for each panel. Panels can be dragged and dropped and rearranged on the dashboard. They can also be resized.
+
+## Move panels
+
+You can drag and drop panels by clicking and holding the panel title, then dragging it to its new location. You can also easily resize panels by clicking the (-) and (+) icons.
+
+![](/img/docs/animated_gifs/drag_drop.gif)
+
+## Tips and shortcuts
+
+* Click the graph title and in the dropdown menu quickly change span or duplicate the panel.
+* Click the Save icon in the menu to save the dashboard with a new name.
```

Comment:
this is not correct either, save icon saves the current dashboard with current name (and so does ctrl/cmd+s). To copy dashboard and save with a new name you have to go into dashboard settings and use the "Save As" button. To just change dashboard title you do that via dashboard settings / General , then save normally.